go/generate.sh

```diff
@@ -22,5 +22,7 @@ protoc \
   --go_opt=paths=source_relative \
   --go-grpc_out=$OUT_DIR \
   --go-grpc_opt=paths=source_relative \
-  "$ROOT/proto/wechaty"/**/*.proto \
```

Comment:
As we have already added the `shopt -s globstar` at the beginning of this script, I believe the `**/*.proto` should have already included all of those files?

Reply:
Yes, I have seen the notes and consulted the relevant information, but I think there are some differences in the details, which did not achieve the expected results.

Reply:
for example `"`, because I need a lot of debugging, I'm lazy 😂

Reply:
I have reviewed the script again, and I believe that the `"$ROOT/proto/wechaty"/**/*.proto` will be equal with your changes below:

```sh
  "$ROOT/proto/wechaty"/puppet/file-box.proto \
  "$ROOT/proto/wechaty"/puppet.proto \
  "$ROOT/proto/wechaty"/puppet/*.proto \
```

Could you please share your insights so that I can understand the reason behind your changes?

Thank you very much.

Reply:
I think `"$ROOT/proto/wechaty"/**/*.proto` cannot load `"$ROOT/proto/wechaty"/puppet.proto`.

Reply:
`/**/*.*` requires that the files in the lower-level directories must be loaded, and `**` will also get all the directories and all the lower-level files.

Reply:
Here's my shell command (Ubuntu Linux, bash)

```sh
$ shopt -s globstar

$ for f in proto/**/*; do echo $f; done
proto/wechaty
proto/wechaty/puppet
proto/wechaty/puppet/base.proto
proto/wechaty/puppet/contact.proto
proto/wechaty/puppet/event.proto
proto/wechaty/puppet/file-box.proto
proto/wechaty/puppet/friendship.proto
proto/wechaty/puppet/location.proto
proto/wechaty/puppet/message.proto
proto/wechaty/puppet/mini-program.proto
proto/wechaty/puppet.proto
proto/wechaty/puppet/room-invitation.proto
proto/wechaty/puppet/room-member.proto
proto/wechaty/puppet/room.proto
proto/wechaty/puppet/tag.proto
proto/wechaty/puppet/url-link.proto
```

Reply:
It seems to be different from what I thought. After I tested it, I found that I didn't change anything, but it looks like CI is ok.